app/assets/stylesheets/sephora_style_guide/variables/_color.scss

```diff
@@ -2,6 +2,7 @@ $brand-primary: #d50032;
 $brand-success: #3a773a;
 $brand-info:    #c000c4;
 $brand-error:   #e00;
+$red-light:     #ff4057;
```

Comment:
Could be `brand-secondary` or `brand-alternate` or something depending on the use case.